src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the interface declaration.